src/marqo/core/validation/validation.py

```diff
@@ -0,0 +1,74 @@
+from enum import Enum
+from decimal import Decimal
+from typing import Optional, Tuple
+from marqo.tensor_search.models.index_settings import IndexSettings
+from pydantic import ValidationError
+from fastapi.responses import JSONResponse
+import json
+
+
+def validate_settings_object(index_name, settings_json) -> Tuple[int, Optional[str]]:
```

Comment:
`IndexManagement` is the right place for this code

Reply:
I think earlier Li asked me to move to new validation folder, but sure I can move it to index_management